simulator:
  enable_physics: false
  sim_rate: 60
  sensor_rate: 1
  control_rate: 40

agent:
  embodied: true
  sensor_height: 0
  image_width: 320
  image_height: 240

position_controller:
  kp_x: 0.5
  kp_y: 0.5
  kp_z: 0.10
  kp_yaw: 0.4
  kd_x: 0.3
  kd_y: 0.3
  kd_z: 0.2
  kd_yaw: 0.1
  max_horz_speed: 1.0
  max_vert_speed: 0.5
  max_yaw_rate_degrees: 10.0
  reached_thresh_xyz: 0.1
  reached_thresh_yaw_degrees: 0.1
